src/views/darkroom.c

```diff
@@ -3660,6 +3660,56 @@ void scrollbar_changed(dt_view_t *self, double x, double y)
   dt_control_navigation_redraw();
 }
 
+float calculate_new_scroll_zoom_tscale (const int up, const gboolean constrained,
```

Comment:
should be a `static` routine and starting with `_` (dt convention): 

static float _calculate_new_scroll_zoom_tscale (const int up, const gboolean constrained,

Reply:
done